### PR TITLE
TINY-10320: Generate css files to make feature compatible with develop

### PR DESCRIPTION
**Pull request by hamza0867**
Related Ticket: TINY-10320

Description of Changes:
* Generate css files to make feature compatible with develop

### CODE REVIEW

File: modules/tinymce/src/themes/silver/main/ts/ui/skin/Loader.ts [hunk 1]
```diff
@@ -53,15 +53,20 @@ const loadShadowDomUiSkins = async (editor: Editor, skinUrl: string): Promise<vo
 };
 
 const loadUrlSkin = async (isInline: boolean, editor: Editor): Promise<void> => {
-  Options.getSkinUrlOption(editor).fold(Fun.noop, (skinUrl) => {
+  Options.getSkinUrlOption(editor).fold(() => {
```
Comment (spocke) [inline modules/tinymce/src/themes/silver/main/ts/ui/skin/Loader.ts:56]: Will this mean that content css will be loaded even if the skin url is not set? It used to be a noop?

**Review by spocke — APPROVED** [review]

File: modules/oxide/gulpfile.js [hunk 2]
```diff
@@ -89,12 +89,25 @@ gulp.task('less', function() {
...
+    .pipe(rename({ extname: '.min.css' }))
+    .pipe(sourcemaps.write('.'))
+    .pipe(gulp.dest('./build/skins'))
+    .pipe(connect.reload());
+});
+
+//
+// Generate JS
+//
+gulp.task('generateJs', function() {
```
Comment (HAFRMO) [inline modules/oxide/gulpfile.js:104]: Why are we adding this to make it compatible with develop? With this won't we generate the css twice and bundling it all up? Seems like it'd take more time and space.

Comment (hamza0867) [reply modules/oxide/gulpfile.js:104]: No we generate them only once

File: modules/tinymce/src/themes/silver/main/ts/ui/skin/Loader.ts (hunk 1 above)
Comment (hamza0867) [reply modules/tinymce/src/themes/silver/main/ts/ui/skin/Loader.ts:56]: No it was not used to be a noop. I just forgot to add this part when the use does not specify a skin url. It is just the default behaviour.

**Pull request merged**